README.md

```diff
@@ -1,6 +1,6 @@
-# team04-w22-5pm-courses
+# team01-s22-6pm-courses
```

Comment:
This is actually `proj01` instead of `team01`, but it's not urgent so you're welcome this later.